server/rules.go

```diff
@@ -54,11 +54,25 @@ func (r *Rules) path(paths ...string) *mux.Route {
 func (r *Rules) pathPrefix(paths ...string) *mux.Route {
 	router := r.route.route.Subrouter()
 	for _, path := range paths {
-		router.PathPrefix(strings.TrimSpace(path))
+		cleanPath := strings.TrimSpace(path)
+		if strings.Contains(cleanPath, "{") {
```

Comment:
Can we leave a note here why `{` is important here?

Reply:
Done :+1: